Makefile.am

```diff
@@ -60,12 +64,17 @@ noinst_HEADERS += contrib/lax_der_parsing.c
 noinst_HEADERS += contrib/lax_der_privatekey_parsing.h
 noinst_HEADERS += contrib/lax_der_privatekey_parsing.c
 
+PRECOMPUTED_LIB = libsecp256k1_precomputed.la
+noinst_LTLIBRARIES = $(PRECOMPUTED_LIB)
+libsecp256k1_precomputed_la_SOURCES =  src/precomputed_ecmult.c src/precomputed_ecmult_gen.c
+libsecp256k1_precomputed_la_CPPFLAGS = $(SECP_INCLUDES)
```

Comment:
@hebasto  This is because we now have a separate compilation unit for the precomputed tables. So Core needs to tell MSVC to build `libsecp256k1_precomputed` separately and link it to the library. 

But I have no idea about the required changes to https://github.com/bitcoin/bitcoin/blob/master/build_msvc/libsecp256k1/libsecp256k1.vcxproj that would be equivalent to this automake snippet.

If this is anyway maintained within the Bitcoin Core org, maybe we should move the vcxproj file to this repo in the future. This will help us add MSVC builds to the CI here.

Reply:
@sipsorcery

Could you look into this?

Reply:
@sipsorcery 

Thanks! It works flawlessly.

Reply:
@sipsorcery That works but it's too much: You don't need the `precompute_` files, the `precomputed` files should be enough. 

Also, this will then recompile the `precomputed_` files every time, which makes recompilation somewhat slower than what we have in our Makefile. (This was the reason why we introduced the `libsecp256k1_precomputed` as a separate object.) But yeah, this only affects compilation time, so it's probably not worth to optimize the `vcxproj` for this.